src/types/verifiableCredentials/index.ts

```diff
@@ -0,0 +1,54 @@
+export interface Credential {
+  context: string[];
+  id: string;
+  type: string[];
+  issuer: string | Issuer;
+  issuanceDate: string;
+  expirationDate?: string;
```

Comment:
@AndrewCLu Does using `timestamps` make sense here?

Reply:
Yes, this is an old version. the new version uses `Date`.

Reply:
@AndrewCLu I meant 
```ts
 const now = new Date(); // date 
 const timestamp = now.getTime() // number
```

Reply:
What would be the benefit here? It would save some space, but not too much since dates are just timestamps under the hood. The nice thing about using `Date` is we are effectively processing Verifiable Credentials the way they are specified throughout the backend - `JSON.Stringify` on a `Date` returns a field compatible with the [standard](https://www.w3.org/TR/vc-data-model/#issuance-date), no extra parsing needed.